spec/acceptance/api/cms_file_spec.rb

```diff
@@ -10,6 +10,7 @@
 
     get '/admin/api/cms/files.:format', action: :index do
       let(:collection) { provider.files }
+      let(:serialized) { representer.send(serialization_format, short: true) }
```

Comment:
If `serialization_format` is not private, we should use `public_send`.

Reply:
Done